.vscode/launch.json

```diff
@@ -11,11 +11,16 @@
             "python": "${workspaceFolder}/lib/python/python.exe",
             "program": "${workspaceFolder}/core/main.py",
             "console": "integratedTerminal",
-            "cwd": "D:\\tests\\sample_site",
+            "cwd": "C:\\Users\\ee102\\Documents\\ez-web-test-site",
```

Comment:
No.

Reply:
This folder should really go in .gitignore

Reply:
Then maybe you can do it here.

Reply:
Then maybe you can do it here.

Reply:
When I’ll get home/be in front of a computer.